metadata.rb

```diff
@@ -17,6 +19,7 @@
 depends          'percona'
 depends          'sysctl'
 depends          'yum'
+depends          'yum-epel'
```

Comment:
What required added EPEL?

Reply:
For server recipe, percona-xtrabackup requires libev package, so I included yum-epel to install libev